IDE/Renesas/e2studio/RX65N/RSK/wolfssl_demo/user_settings.h

```diff
@@ -263,3 +270,6 @@
 
 /*-- strcasecmp */
 #define XSTRCASECMP(s1,s2) strcmp((s1),(s2))
+
+/* use original ASN parsing */
+#define WOLFSSL_ASN_ORIGINAL
```

Comment:
Why use the older ASN method? Is the new ASN template not supported or not working?

Reply:
When using TSIP TLS, it requires key positions in a cert. It is done by RsaPublicKeyDecodeRawIndex() in asn.c and the functions still use older functions such as SkipObjectId().

Reply:
Can you please investigate updating support for ASN template?

Reply:
ok,  I will.